### PR TITLE
Tracks Cargo.lock

### PR DESCRIPTION
**Pull request by sr-gi**
Rationale: https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html

### CODE REVIEW

**Review by mariocynicys — APPROVED** [review]
I'm sold!

**Comment by mariocynicys** [comment]
from `cargo help install`:
```man
       By default, the Cargo.lock file that is included with the package will be ignored. This means that Cargo will recompute which versions
       of dependencies to use, possibly using newer versions that have been released since the package was published. The --locked flag can
       be used to force Cargo to use the packaged Cargo.lock file if it is available. This may be useful for ensuring reproducible builds, to
       use the exact same set of dependencies that were available when the package was published. It may also be useful if a newer version of
       a dependency is published that no longer builds on your system, or has other problems. The downside to using --locked is that you will
       not receive any fixes or updates to any dependency. Note that Cargo did not start publishing Cargo.lock files until version 1.37,
       which means packages published with prior versions will not have a Cargo.lock file available.
```
TL;DR: `cargo install` will ignore `Cargo.lock` by default unless `--locked` flags was passed.

**Comment by sr-gi** [comment]
> from `cargo help install`:
> 
> ```
>        By default, the Cargo.lock file that is included with the package will be ignored. This means that Cargo will recompute which versions
>        of dependencies to use, possibly using newer versions that have been released since the package was published. The --locked flag can
>        be used to force Cargo to use the packaged Cargo.lock file if it is available. This may be useful for ensuring reproducible builds, to
>        use the exact same set of dependencies that were available when the package was published. It may also be useful if a newer version of
>        a dependency is published that no longer builds on your system, or has other problems. The downside to using --locked is that you will
>        not receive any fixes or updates to any dependency. Note that Cargo did not start publishing Cargo.lock files until version 1.37,
>        which means packages published with prior versions will not have a Cargo.lock file available.
> ```
> 
> TL;DR: `cargo install` will ignore `Cargo.lock` by default unless `--locked` flags was passed.

Yep, last rebase already included `--locked`

**Comment by sr-gi** [comment]
close #148
